packages/engine/src/simulation/package/state/packages/behavior_execution/mod.rs

```diff
@@ -108,7 +108,7 @@ impl PackageCreator for Creator {
             .index_of(behavior_index_col.value())?;
 
         Ok(Box::new(BehaviorExecution {
-            behavior_map: Arc::clone(self.get_behavior_map()?),
+            _behavior_map: Arc::clone(self.get_behavior_map()?),
```

Comment:
Ditto for these ,if they're unused, no need to store them. Unless there's something to do with them being dropped early or whatever, in which case definitely needs a comment. My guess is that they're unneeded in the struct